robot.yaml

```diff
@@ -0,0 +1,22 @@
+# For more details on the format and content:
+# https://github.com/robocorp/rcc/blob/master/docs/recipes.md#what-is-in-robotyaml
+
+tasks:
+  Minimal Task:
```

Comment:
Rename this to something descriptive, as it'll be the one visible in Control Room

Reply:
I left it as it is for now, as we start from the basic template and do not want to touch too many files. Might modify it though